recipes/cuda-command-line-tools/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "12.0.0" %}
+
+package:
+  name: cuda-command-line-tools
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_documentation/LICENSE.txt
+  sha256: 5db25d4fd138013b563f9a3d1d87f7de7df1dac014fd4cccdfbb3435a5cff761
+
+build:
+  number: 0
+  skip: true  # [osx or win]
+
+requirements:
+  run:
+    - cuda-cupti-dev 12.0.90
+    - cuda-gdb 12.0.90
+    - cuda-nvdisasm 12.0.76
+    - cuda-nvprof 12.0.90
+    - cuda-nvtx 12.0.76
+    - cuda-sanitizer-api 12.0.90
+
+test:
+  commands:
+    - exit 0
+
+about:
+  home: https://developer.nvidia.com/cuda-toolkit
+  license_file: LICENSE.txt
+  license: LicenseRef-NVIDIA-End-User-License-Agreement
+  license_url: https://docs.nvidia.com/cuda/eula/index.html
+  summary: Meta-package containing the command line tools to debug CUDA applications
```

Comment:
I'm assuming that this is copy-pasted from NVIDIA docs? Otherwise I'd recommend aligning on the unhyphenated spelling of metapackage.

Reply:
Yes the summaries of all formats have used "Meta-package", although I agree that "metapackage" would be better.